docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -6652,9 +6653,18 @@ SELECT [ "[" ANY N "]" | "[" FIRST N "]" ] [ALL | DISTINCT] select-list
       [,colname [ASC[ENDING] | DESC[ENDING]]]...]
    [GROUP BY {colname | colnum} [,{colname | colnum}]...]
    [HAVING search-condition]
-   [access-clause ]
+   [access-clause]
    [mode-clause]
 
+With-clause is:
+with_clause_elements

```

Comment:
I think the keyword "WITH" is missing here. Shouldn't it say With-clause is: WITH with_clause_elements